src/query/ee/tests/it/aggregating_index/index_refresh.rs

```diff
@@ -305,26 +305,39 @@ async fn test_refresh_agg_index_with_limit() -> Result<()> {
     Ok(())
 }
 
-#[ignore = "flaky"]
 #[tokio::test(flavor = "multi_thread")]
 async fn test_sync_agg_index() -> Result<()> {
+    test_sync_agg_index_after_insert().await?;
+    test_sync_agg_index_after_copy_into().await?;
+
+    Ok(())
+}
+
+async fn test_sync_agg_index_after_insert() -> Result<()> {
     let (_guard, ctx, root) = create_ee_query_context(None).await.unwrap();
     ctx.get_settings()
         .set_enable_refresh_aggregating_index_after_write(true)?;
     let fixture = TestFixture::new_with_ctx(_guard, ctx).await;
-
+    let ctx = fixture.ctx();
+    execute_sql(ctx.clone(), "DROP TABLE IF EXISTS t0").await?;
     // Create table
     execute_sql(
-        fixture.ctx(),
+        ctx.clone(),
         "CREATE TABLE t0 (a int, b int, c int) storage_format = 'parquet'",
     )
     .await?;
 
     // Create agg index `index0`
     let index_name = "index0";
 
+    execute_sql(
+        ctx.clone(),
+        &format!("DROP AGGREGATING INDEX IF EXISTS {}", index_name),
+    )
+    .await?;
```

Comment:
Do we need to drop the  table and the index? I think `create_ee_query_context` will create a brand new environment.

Reply:
fixed